### DISCUSSION TITLE
Collect stats during linting

### DISCUSSION THREAD

**Discussion by cidrblock** [ansible/ansible-lint#3445] [Ideas]
For a given project, collect stats about tank count, include, plguins used etc.

These can later be used for:
- test requirements used in run time
- untested plugins e.g. coverage

Anything else we could do here? Thoughts?

**Comment by MarkusTeufelberger** [comment]
Getting a "linter coverage" report would already be interesting, since it is not always clear if some tasks file that gets only dynamically included somewhere is actually being linted.
